### PR TITLE
feat: add `math/base/special/acosd`

### LINKED ISSUE #36: Implement `acosd`
Compute the arccosine of `x` where the output is in degrees. `@stdlib/math/base/special/acosd`.

### PR DESCRIPTION
**Pull request by saisrikardumpeti**
Resolves #36 

## Description

> What is the purpose of this pull request?

This pull request:

-  Adds functionality to compute `acosd` of any given input.
- Say we have input `x`, so it should satisfies this `x <= 1` or `x >= -1` else it will return `NaN` 

| input | acosd |
| -------- | ------- |
| 0  | 90    |
| 0.5 |  60   |
| $\frac{1}{\sqrt{2}}$ | 45    |
| $\frac{\sqrt{3}}{2}$ | 30 |
| 1 | 0 |

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #36
-   fixes #

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by saisrikardumpeti** [comment]
@Planeshifter this PR is ready for review.

File: lib/node_modules/@stdlib/math/base/special/acosd/docs/repl.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+{{alias}}( x )
+    Computes the arccosine of a double-precision floating-point number.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/acosd/docs/repl.txt]:
```suggestion
    Computes the arccosine (in degrees) of a double-precision floating-point 
    number.
```

File: lib/node_modules/@stdlib/math/base/special/acosd/package.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+  },
+  "contributors": [
+    {
+      "name": "The Stdlib Authors",
+      "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+    }
+  ],
+  "main": "./lib",
+  "gypfile": true,
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/acosd/package.json]: Should not be set when there is no C implementation / native add-on.

```suggestion
```

File: lib/node_modules/@stdlib/math/base/special/acosd/package.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    "stdlib",
+    "stdmath",
+    "mathematics",
+    "math",
+    "math.asin",
+    "asin",
+    "degree",
+    "arccosine",
+    "sine",
+    "inverse",
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/acosd/package.json]:
```suggestion
    "degree",
    "arccosine",
    "cosine",
    "inverse",
```

**Review by Planeshifter — APPROVED** [review]

**Comment by Planeshifter** [comment]
Will merge after CI has passed; thank you for getting this PR over the finish line!

**Pull request merged**